ci/linkcheck.sh

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env bash
 
-set -ev
+set -e
 set -o pipefail
 
 # https://docs.travis-ci.com/user/environment-variables/#default-environment-variables
 if [ "$TRAVIS_EVENT_TYPE" = "cron" ] ; then # running in cron job
   FLAGS=""
 
   echo "Doing full link check."
+  set -x
```

Comment:
Not sure I understand this change. What does the `-x` flag do? Shouldn't this be `-v` to keep the old behaviour in CI?

Reply:
`-x` means 'expand all variables and command substitutions before echoing them'. Here's with -v:

```
Doing full link check.

exec mdbook-linkcheck $FLAGS -- $TRAVIS_BUILD_DIR
```
and with -x:
```
Doing full link check.
+ exec mdbook-linkcheck --
```
so you can tell that it's checking without additional arguments, where you couldn't before. It's even more helpful when only some files were changed:
```
$ CI=true TRAVIS_COMMIT_RANGE=HEAD~2 ci/linkcheck.sh
Checking files changed in HEAD~2: README.md ci/linkcheck.sh src/building/bootstrapping.md 
+ exec mdbook-linkcheck --no-cache -f README.md ci/linkcheck.sh src/building/bootstrapping.md --
```

That said, since the script manually echoes the commands it's about to execute, it's not _super_ helpful, but it might still be nice for finding bugs.